src/components/FactualConsistencyBadge.tsx

```diff
@@ -7,9 +7,7 @@ interface Props {
 export const FactualConsistencyBadge = ({ score }: Props) => {
   let badge;
 
-  if (score === undefined) {
-    badge = <VuiBadge color="accent">Calculating Factual Consistency Score…</VuiBadge>;
```

Comment:
I think we should keep this so that the user knows the FCS is enabled and the UI is pulling it.
If streaming has ended and FCS score still hasn't been found, then we can show the empty state you have below.

Reply:
we did removed that from the vectara-answer thats why I have removed from that chatbot as well.

Reply:
@adeelehsan I don't think we should couple this project or other projects to Vectara Answer. It's fine that you removed the FCS loading state there, but I agree with Deryk that we should keep it here.

Reply:
sure. I'll revert it.